framework/chi_runtime.cc

```diff
@@ -157,11 +163,13 @@ Chi::run_time::ParseArguments(int argc, char** argv)
 
   } // for argument
 
+#ifdef OPENSN_WITH_LUA
   if (Chi::run_time::dump_registry_)
   {
     ChiObjectFactory::GetInstance().DumpRegister();
     Chi::console.DumpRegister();
   }
+#endif
 }
 
 // ############################################### Initialize ChiTech
```

Comment:
another mention

Reply:
this is a .cc file. it can be done later

Reply:
We talked about this [here](https://github.com/Open-Sn/openSn/issues/3#issuecomment-1795912565). So I assume a PR where we just fix the comment style first. And another one later with actually changing the names...